.goreleaser.yml

```diff
@@ -22,10 +33,10 @@ nfpm:
     - rpm
 
 brew:
-  name: oscalkit
+  name: compliance-masonry
   github:
     owner: opencontrol
-    name: homebrew-oscalkit
+    name: homebrew-compliance-masonry
   commit_author:
     name: OpenControl
     email: dev@open-control.org
```

Comment:
Does this EMail exist?

Reply:
@shawndwells figured you would know more than I about that.

Reply:
Not something that's being changed in the PR, so no reason to hold a merge, just caught my eye.